common/src/main/java/org/wso2/testgrid/common/util/StringUtil.java

```diff
@@ -66,4 +69,20 @@ public static String generateRandomString(int characterCount) {
         }
         return salt.toString();
     }
+
+    /**
+     * Generate a string which include all the properties (key and the value).
+     * @param properties Properties
+     * @return List of properties (key and value) as a String
+     */
+    public static String getPropertiesAsString(Properties properties) {
+        if (properties.isEmpty()) {
+            return "Empty property-list.";
```

Comment:
Shouldn't we return an empty string in this case ?

Reply:
+1, The method has been updated. Thanks for pointing out.